books/refgram/src/grammar/annotations.md

```diff
@@ -36,7 +36,7 @@ could be compared to emojis or lojban's attitudinals, like `de gli` (love/❤️
 
 __dei__ uses this predicate as a metadata tag or _#hashtag_.
 
-## Free parenthetical DA...DAI (suffix)
+## Free parenthetical da...dai (suffix)
```

Comment:
Families are written in uppercase, even if they have one member. If in the future there are more members they'll follow the same grammar. However it would indeed be better in the text bellow to indicate what are those single members.